requirements.txt

```diff
@@ -1 +1,24 @@
-boto3==1.9.111
\ No newline at end of file
+apiclient==1.0.3
+boto3==1.9.111
+botocore==1.12.108
+cachetools==3.1.0
+certifi==2019.3.9
+chardet==3.0.4
+docutils==0.14
+google-api-python-client==1.7.8
+google-auth==1.6.3
+google-auth-httplib2==0.0.3
+httplib2==0.12.1
+idna==2.8
+jmespath==0.9.4
+oauth2client==4.1.3
+pyasn1==0.4.5
+pyasn1-modules==0.2.4
+python-dateutil==2.8.0
+python-lambda-local==0.1.8
+requests==2.21.0
+rsa==4.0
+s3transfer==0.2.0
+six==1.12.0
```

Comment:
why? we're in python 3, who's using six?

Reply:
the google python api client for some reason